scene/3d/skeleton_3d.cpp

```diff
@@ -540,6 +540,15 @@ void Skeleton3D::set_bone_name(int p_bone, const String &p_name) {
 	bones.write[p_bone].name = p_name;
 }
 
+String Skeleton3D::get_concatenated_bone_names() const {
```

Comment:
If this is only needed in the mapper, I would probably put it over there and not add more stuff to Skeleton.

Reply:
Okay, I removed it from Skeleton3D.